### ISSUE TITLE
Support for german language in user interface

### ISSUE THREAD

**Issue by OliverIckler** [athombv/homey#2698]
**Metadata**

* **Homey Version**: latest
* **Homey Log Code**:
* **Smartphone Platform**: IOS
* **Smartphone App Version**: latest

**If related to a Homey App:**
* **Homey App Version**:
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**:

**What did you try to achieve?**

I am writing a new driver for my devolo components.
Why is there only en and nl supported in user interface?
I want to add de settings also.

**What did you expect as result?**

German Language when using Homey

**What was the result?**

English usage :-(

**Any other remarks**

**Comment by OliverIckler** [comment]
If you need someone to translate the UI, please contact me, so i will do it.

**Comment by Mguss** [comment]
Hi. Is it possible to get code for language in user interface and translate it by myself to Swedish?

**Comment by OliverIckler** [comment]
First of all we translate it to german. I asked before you ;-)

**Comment by Mguss** [comment]
What I was thinking of was if there was an opportunity to make the translation into Swedish by me. Is the code available for that?

**Comment by OliverIckler** [comment]
I don't think so. I want to do the same for german language. But currently there is only EN and NL available in the home settings.
